server/sublist.go

```diff
@@ -83,7 +83,7 @@ func NewSublist() *Sublist {
 // Insert adds a subscription into the sublist
 func (s *Sublist) Insert(sub *subscription) error {
 	// copy the subject since we hold this and this might be part of a large byte slice.
-	subject := string(sub.subject)
+	subject := (" " + string(sub.subject))[1:]
```

Comment:
I think there is a better way. Maybe just leave as a cast for now with comment, or maybe use bytes package somehow? We could also do append to nil string from []byte I think.